scripts/deploy-transport-controller.sh

```diff
@@ -1,112 +0,0 @@
-#!/usr/bin/env bash
-# Copyright 2024 The KubeStellar Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
```

Comment:
are we sure we want to delete this file?
until there's a different way to deploy a locally built image (e.g., for e2e purpose or just for testing my changes), I wouldn't remove it.